handbook/product/onboarding/index.md

```diff
@@ -15,13 +15,17 @@ Welcome to Sourcegraph! As a member of the product team, it is your job to be th
 ## Product team member checklist
 
 - Get to know our team and learn about Sourcegraph (company and product)
+  - Schedule 30-min 1:1s with your manager for the first three days at Sourcegraph to keep up with your onboarding and ask all the questions you might have
 - Finish [general onboarding tasks](../../people-ops/onboarding/index.md#for-all-new-teammates)
   - Set up your email filters, especially for GitHub and feedback
 - Get to know the product
   - Complete of the [Engineering onboarding tasks](../../engineering/onboarding/index.md). Almost all of these are relevant to you.
     - You will need to run Sourcegraph locally to test/validate work that engineering is doing, to provide early/often feedback.
   - Review [product resources](../index.md#resources)
   - [Products](https://about.sourcegraph.com/product)
+  - Watch this [demo video](https://drive.google.com/file/d/1idbCnce5MIvtAV0GOOwgB68zQJB2WmZ9/view)
```

Comment:
Perhaps add "Product Demo from Customer Engineering" for more context.

Reply:
```suggestion
  - Learn how the Customer Engineering team gives demos and talks about the product in the [product demo recording](https://drive.google.com/file/d/1idbCnce5MIvtAV0GOOwgB68zQJB2WmZ9/view).
```